src/main/scala/com/fulcrumgenomics/bam/FilterBam.scala

```diff
@@ -51,6 +52,11 @@ import htsjdk.samtools.util.IntervalList
      |marked as secondary alignments, reads marked as duplicates, and if a set of Intervals are provided,
      |reads that do not overlap any of the intervals.
      |
+     |The `--remove-unpaired-reads`, which removes reads that are not part of a mapped pair will remove
```

Comment:
I'll vote for anything that isn't `--remove-unpaired-reads` since I know in "Broad"-speak that "unpaired" meant a read pair where not both ends are mapped, but I know other folks use unpaired differently (unpaired means single-end reads, or where exactly one end maps).

By the way, not all options are `--remove-*`, see `minMapQ`!

How about two options:
1. `--remove-fragment-reads`
2. `--require-mapped-pairs`
?